src/tests/system/tests/test_proxy.py

```diff
@@ -14,22 +14,24 @@
 
 @pytest.mark.topology(KnownTopology.LDAP)
 @pytest.mark.ticket(bz=895570)
-def test_invalid_attribute_syntax(client: Client, ldap: LDAP):
+def test_proxy__nslcd_fast_alias_set_to_true_with_no_ldb_modify_errors(client: Client, ldap: LDAP):
     """
-    :title: Enabling proxy_fast_alias shows "ldb_modify failed:
-        [Invalid attribute syntax]" for id lookups.
+    :title: Enabling proxy_fast_alias should not show "ldb_modify failed: [Invalid attribute syntax]" for id lookups.
     :setup:
         1. Setup sssd for proxy provider.
         2. Enable proxy_fast_alias.
         3. Setup nslcd services.
-        4. Add Ou and User.
+        4. Add OU and User.
     :steps:
         1. id lookup a user.
         2. Check logs for "ldb_modify failed".
     :expectedresults:
         1. id look up should success.
         2. Errors should not be seen on enabling proxy_fast_alias.
     :customerscenario: True
+    :TODO: I don't think this test really adds any value. It's a non functional test, if people think it adds value.
+           We should make it a functional test and check the ldb cache for the attribute. Or move this to
+           test_logging.py and have this be a logging test.
     """
     client.sssd.config["domain/test"] = {
```

Comment:
sssd.common.proxy should be used instead of manual configuration. Could you please take care of it as well? Same in the other test.